packages/eslint-config-wpcalypso/index.js

```diff
@@ -95,55 +95,57 @@ module.exports = {
-		'space-in-parens': [ 2, 'always' ],
-		'space-infix-ops': [ 2, { int32Hint: false } ],
+		'space-in-parens': [ 'error', 'always' ],
+		'space-infix-ops': [ 'error', { int32Hint: false } ],
 		'space-unary-ops': [
-			2,
+			'error',
 			{
 				overrides: {
 					'!': true,
 				},
 			},
 		],
 		// Assumed by default with Babel
-		strict: [ 2, 'never' ],
-		'template-curly-spacing': [ 2, 'always' ],
+		strict: [ 'error', 'never' ],
+		'template-curly-spacing': [ 'error', 'always' ],
 
 		// jsdoc plugin
-		'jsdoc/check-access': 2,
-		'jsdoc/check-alignment': 2,
-		'jsdoc/check-param-names': 2,
-		'jsdoc/check-tag-names': 0,
-		'jsdoc/check-types': 2,
-		'jsdoc/check-values': 2,
-		'jsdoc/empty-tags': 2,
-		'jsdoc/implements-on-classes': 2,
-		'jsdoc/newline-after-description': 2,
-		'jsdoc/no-undefined-types': 2,
-		'jsdoc/require-jsdoc': 0,
-		'jsdoc/require-param': 2,
-		'jsdoc/require-param-description': 2,
-		'jsdoc/require-param-name': 2,
-		'jsdoc/require-param-type': 0,
-		'jsdoc/require-returns': 0,
-		'jsdoc/require-returns-check': 2,
-		'jsdoc/require-returns-description': 2,
-		'jsdoc/require-returns-type': 0,
-		'jsdoc/valid-types': 0,
+		'jsdoc/check-access': 'error',
+		'jsdoc/check-alignment': 'error',
+		'jsdoc/check-param-names': 'error',
+		'jsdoc/check-tag-names': 'off',
+		'jsdoc/check-types': 'error',
+		'jsdoc/check-values': 'error',
+		'jsdoc/empty-tags': 'error',
+		'jsdoc/implements-on-classes': 'error',
+		'jsdoc/newline-after-description': 'error',
+		'jsdoc/no-undefined-types': 'error',
+		'jsdoc/require-jsdoc': 'off',
+		'jsdoc/require-param': 'error',
+		'jsdoc/require-param-description': 'error',
+		'jsdoc/require-param-name': 'error',
+		'jsdoc/require-param-type': 'off',
+		'jsdoc/require-returns': 'off',
+		'jsdoc/require-returns-check': 'error',
+		'jsdoc/require-returns-description': 'error',
+		'jsdoc/require-returns-type': 'off',
+		'jsdoc/valid-types': 'off',
 
 		// wpcalypso plugin
-		'wpcalypso/i18n-ellipsis': 2,
-		'wpcalypso/i18n-no-collapsible-whitespace': 2,
-		'wpcalypso/i18n-no-variables': 2,
-		'wpcalypso/i18n-no-placeholders-only': 2,
-		'wpcalypso/i18n-no-this-translate': 2,
-		'wpcalypso/i18n-mismatched-placeholders': 2,
-		'wpcalypso/i18n-named-placeholders': 2,
-		'wpcalypso/import-docblock': 2,
-		'wpcalypso/jsx-gridicon-size': 2,
-		'wpcalypso/jsx-classname-namespace': 2,
-		'wpcalypso/redux-no-bound-selectors': 2,
-		yoda: 0,
+		'wpcalypso/i18n-ellipsis': 'error',
+		'wpcalypso/i18n-no-collapsible-whitespace': 'error',
+		'wpcalypso/i18n-no-variables': 'error',
+		'wpcalypso/i18n-no-placeholders-only': 'error',
+		'wpcalypso/i18n-no-this-translate': 'error',
+		'wpcalypso/i18n-mismatched-placeholders': 'error',
+		'wpcalypso/i18n-named-placeholders': 'error',
+		'wpcalypso/import-docblock': 'error',
+		'wpcalypso/jsx-gridicon-size': 'error',
+		'wpcalypso/jsx-classname-namespace': 'error',
+		'wpcalypso/redux-no-bound-selectors': 'error',
+		'wpcalypso/wp-no-unsafe-features': 'error',
```

Comment:
This line adds the new rule. The rest just rewrite from confusing numbers to clear strings.